osi_common.proto

```diff
@@ -215,6 +215,39 @@ message Identifier
     optional uint64 value = 1;
 }
 
+// \brief References to external objects
+//
+// The external reference is used to identify objects defined in non-OSI
+// descriptions. This could be other OpenX-Standards as well as user defined
+// simulation environments.
+//
+message ExternalReference
+{
+    // The source of the external references
+    //
+    // Defines the original source of an object as an URI.
+    // The URI should be follow the syntax according to 
+    // \link https://tools.ietf.org/html/rfc3986 RFC 3986\endlink.
+    //
+    optional string reference = 1;
+    
+    // The type of the external references
+    //
+    // Can be used to describe the original source, e.g. OpenDRIVE 1.6
+    //
+    optional string type = 2;    
```

Comment:
Insert list of predefined types (like those for OpenX) and general rules/guidance for user-defined standards or objects.

Reply:
**Output CCB 12.05.2021:**
1. Remove the version from the comment description, name should be enough (OpenDRIVE instead of OpenDrive 1.6)
2. Use the domain name notation, e.g. **net.asam.opendrive** instead of **OpenDRIVE**.
3. Indicate that the type field should be lower case.